src/command_modules/azure-cli-servicebus/HISTORY.rst

```diff
@@ -3,9 +3,14 @@
 Release History
 ===============
 
+0.3.1
++++++
+* Added Read-Only migration_state to MigrationConfigProperties to show current Service Bus Standard to Premium namespace migration state.
+
 0.3.0
 +++++
 * BREAKING CHANGE: 'list' commands errors for resource(s) NotFound(404) are now handled in the typical way instead of showing empty list
+* Added Read-Only migration_state to MigrationConfigProperties to show current Service Bus Standard to Premium namespace migration state.
```

Comment:
This should only be in one version or the other.

Reply:
updated the history file for only  - * Added Read-Only migration_state